contracts/UniswapV2Pair.sol

```diff
@@ -88,15 +96,16 @@ contract UniswapV2Pair is IUniswapV2Pair, UniswapV2ERC20 {
     // if fee is on, mint liquidity equivalent to 1/6th of the growth in sqrt(k)
```

Comment:
This comment should be updated to reflect a dynamic protocolFee and dynamic swap fee.  That is, it may not be equivalent to 1/6th the growth in sqrt(k)